seed/challenges/01-responsive-web-design/applied-visual-design.json

```diff
@@ -671,9 +672,9 @@
         "<div class=\"fullCard\">",
         "  <div class=\"cardContent\">",
         "    <div class=\"cardText\">",
-        "      <strong><h4><del>Google</del>Alphabet</h4></strong>",
+        "      <h4><del>Google</del>Alphabet</h4>",
         "      ",
-        "      <em><p>Google was founded by Larry Page and Sergey Brin while they were Ph.D. students at Stanford University.</p></em>",
+        "      <em><p>Google was founded by Larry Page and Sergey Brin while they were Ph.D. students at <strong>Stanford University</strong>.</p></em>",
```

Comment:
I tested all the subsequent challenges and I didn't see it failing. Let me verify this. Thanks.